lib/src/js/polymer_base_dart.html

```diff
@@ -19,6 +19,30 @@
       if (this.__dartDetached__) this.__dartDetached__();
     };
 
+    var originalDeserialize = Polymer.Base.deserialize;
```

Comment:
Actually I think the __dartAttached__ and __dartDetached__ are actually dead code? I don't see us assigning it anywhere.